mono/utils/mono-tls.h

```diff
@@ -55,7 +55,15 @@ g_static_assert (TLS_KEY_DOMAIN == 0);
 #define mono_native_tls_alloc(key,destructor) ((*(key) = TlsAlloc ()) != TLS_OUT_OF_INDEXES && destructor == NULL)
 #define mono_native_tls_free TlsFree
 #define mono_native_tls_set_value TlsSetValue
-#define mono_native_tls_get_value TlsGetValue
+
+static inline LPVOID mono_native_tls_get_value (MonoNativeTlsKey key)
```

Comment:
1. Cosmetic: “L” is a 16bit-ism. Prefer modern “PVOID”.
2. Use __declspec(thread) under “#if _MSC_VER” it is tremendously more efficient, like 3 cheap instructions, no lock, no call.

Reply:
1. I used exactly what is in the documentation.
2. That is already done elsewhere. This code will not get hit if __declspec(thread) is supported.